src/libraries/System.Dynamic.Runtime/README.md

```diff
@@ -0,0 +1,8 @@
+# System.Dynamic.Runtime package
```

Comment:
package here is a misnomer: we don't produce a package for this.  It's only in the shared framework.  Should this say "library" instead?  Same applies to System.Linq.Expressions.